exec/java-exec/src/main/java/org/apache/drill/exec/store/pcap/PcapFormatConfig.java

```diff
@@ -17,12 +17,25 @@
  */
 package org.apache.drill.exec.store.pcap;
 
+import com.fasterxml.jackson.annotation.JsonInclude;
 import com.fasterxml.jackson.annotation.JsonTypeName;
 import org.apache.drill.common.logical.FormatPluginConfig;
+import org.apache.drill.shaded.guava.com.google.common.collect.ImmutableList;
 
-@JsonTypeName("pcap")
+import java.util.List;
+
+@JsonTypeName(PcapFormatPlugin.PLUGIN_NAME)
 public class PcapFormatConfig implements FormatPluginConfig {
 
+  private static final List<String> DEFAULT_EXTS = ImmutableList.of(PcapFormatPlugin.PLUGIN_NAME);
+
+  public List<String> extensions;
```

Comment:
Since you have added extensions, consider updating hadCode and equals methods.

Reply:
Done